tasking/pulp/tasking/services/worker_watcher.py

```diff
@@ -100,3 +102,43 @@ def handle_worker_offline(event):
     msg = _("Worker '%(worker_name)s' shutdown") % event_info
     _logger.info(msg)
     delete_worker(event_info['worker_name'], normal_shutdown=True)
+
+
+def delete_worker(name, normal_shutdown=False):
+    """
+    Delete the :class:`~pulp.app.models.Worker` with name from the database, cancel any associated
```

Comment:
I believe autodoc expects the short description to be 1 line. I relies on that because that is what the docstring pep requires. Can this be shortened to 1 line < 100 characters somehow?